giraffe/src/components/AnnotationLine.tsx

```diff
@@ -120,7 +120,7 @@ export const AnnotationLine: FunctionComponent<AnnotationLineProps> = props => {
           ${clampedStart - pixelMargin}, ${RANGE_HEIGHT}`,
       fill: color,
       id: props.id,
-      style: {cursor: 'pointer'},
+      style: {cursor: 'pointer', opacity: '60%'},
```

Comment:
Why 60%? Can we make this into a constant somewhere for easier reference and self documentation?

Reply:
that is what looks good; you can see the line while the 100% triangle shows up.  and @daviddkkim  approved it.

Reply:
i will add documentation to explain it.